bpmn2stamp-converter/pom.xml

```diff
@@ -201,39 +201,36 @@
                 </configuration>
             </plugin>
             <!-- JOPA CLASS GENERATOR -->
-<!--            <plugin>-->
-<!--                &lt;!&ndash; Generate entity classes from OWL integrity constraints &ndash;&gt;-->
-<!--                <groupId>cz.cvut.kbss.jopa</groupId>-->
-<!--                <artifactId>jopa-maven-plugin</artifactId>-->
-<!--                <version>${cz.cvut.kbss.jopa.version}</version>-->
-<!--                <configuration>-->
-<!--                    &lt;!&ndash; STAMP &ndash;&gt;-->
+            <plugin>
+                <!-- Generate entity classes from OWL integrity constraints -->
+                <groupId>cz.cvut.kbss.jopa</groupId>
+                <artifactId>jopa-maven-plugin</artifactId>
+                <version>${cz.cvut.kbss.jopa.version}</version>
+                <configuration>
+                    <!-- Uncomment one of the following sections (only one at a time) to generate the corresponding model -->
+                    <!-- STAMP -->
 <!--                    <package>model.stamp</package>-->
 <!--                    <ontology-iri>http://onto.fel.cvut.cz/ontologies/stamp</ontology-iri>-->
-<!--                    &lt;!&ndash; BBO &ndash;&gt;-->
+                    <!-- BBO -->
```

Comment:
move this to execution

Reply:
moved to multiple executions for every model (bbo, stamp, bbo-e)